composer-require-checker.json

```diff
@@ -0,0 +1,8 @@
+{
+  "symbol-whitelist" : [
+    "null", "true", "false",
+    "static", "self", "parent",
+    "array", "string", "int", "float", "bool", "iterable", "callable", "void", "object",
+    "OpenSSLAsymmetricKey"
```

Comment:
This is a PHP8 thing.
I've run the CI job on 7.4 to be aligned with the other jobs, once everything will be on PHP8 too this can be dropped